README.md

```diff
@@ -261,6 +262,22 @@ Generate token to replace in the project.
 
 ## Deploy application
 
+### Deploy with docker compose
+
+* build create jar: `./gradlew clean bootJar`
+* Start docker compose
+
+```shell
+docker-compose -f docker/docker-compose.yml up --build
+```
+
+* Stop docker compose
+
+```shell
+cd docker
+docker-compose down
+```
+
 ### Deploy with docker
```

Comment:
Here I would add that if the application is started  using this scenario, the database should be started using command line via:
`docker run --name surrealdb -p 8000:8000 -d surrealdb/surrealdb:latest start`

Reply:
Just a minor comment,
docker-compose ( has been replaced with new command in v2 docker )
v2
docker compose ( without hyphen)

Reply:
@nverbic about this comment: 

> database should be started using command line via

it is covered in the docker compose here:

docker compose -f docker/docker-compose.yml up --build

Reply:
What about adding this info after the line 167, after: 

* Start Spring Boot Application:
```shell
./gradlew bootRun
```
if app is started outside of the docker?

Reply:
I added your suggestion when starting locally without the docker compose. Thanks for the suggestion.